draft-pauly-add-resolver-discovery.md

```diff
@@ -89,33 +89,38 @@ they appear in all capitals, as shown here.
 
-Direct Resolver:
-: A DNS resolver using any transport that is provisioned directly by a local router or a VPN.
+Companion DoH Server:
+: A DNS resolver that provides connectivity over HTTPS (DoH) that is designated as 
+equivalent to querying a particular Direct Resolver.
 
 # Designated Resolvers
 
-- Based on DNSSEC-signed SVCB record
-- Based on confirmation of a trusted domain, listed in additional information
-- Use IP hints for addresses
+An encrypted DNS resolver, such as a DoH or DoT server, can be designated for use in resolving names within one or more zones. This means that clients can learn about an explicit mapping from a given domain or zone to one or more Designated Resolvers, and use that mapping to select the best resolver for a given query.
+
+Designating a resolver MUST rely on mutual agreement between the entity managing a zone (the Domain Owner) and the entity operating the resolver. These entities can be one and the same, or a Domain Owner can choose to designate a third-party resolver to handle its traffic. By presenting proof of this mutual agreement, the two entities assert to clients that interacting with the designated resolver will not expose information about names being resolved to an entity that would otherwise not be able to learn such information.
+
+As an example with only one entity, a company that runs many sites within "enterprise.example.com" can provide its own DoH resolver, "doh.enterprise.example.com", and designate only that resolver for all names that fall within "enterprise.example.com". This means that no other resolver would be designated for those names, and clients would only be resolving names with the same entity that would receive TLS connections.
+
+As an example with several entities, the organization that runs sites within "example.org" may work with two different Content Delivery Networks (CDNs) to serve its sites. It might designate names under "example.com" to two different entities, "doh.cdn-a.net" and "doh.cdn-b.net". These are CDNs that have an existing relationship with the organization that runs "example.org", and have agreements with that organization about how data about names and users is handled.
 
-# Discovering Designated DoH Resolvers
+There are several methods that can be used to designate a resolver:
 
-Clients dynamically build and maintain a set of known Designated DoH Servers. The information
-that is associated with each server is:
+- Based on SVCB DNS records issued to another resolver ({{svcb}})
+- Based on information from Designated DoH Resolver that is confirmed via SVCB DNS records ({{pvd}})
+- Based on mutual confirmation of domains over HTTPS ({{pvd-mutual}})
 
-- The URI Template of the DoH server {{!RFC8484}}
-- A list of domains for which the DoH server is designated
+Note that clients SHOULD NOT accept designations for effective top-level domains (eTLDs), such as ".com".
```

Comment:
MUST NOT? Is there a case where a client might want to ignore this?

Reply:
Done